content/hardware/04.pro/boards/portenta-x8/tutorials/02.x8-fundamentals/portenta-x8-fundamentals.md

```diff
@@ -1,85 +1,233 @@
 ---
-title: '02. How to Use the Portenta X8 Manager'
+title: '02. Portenta X8 Fundamentals & Manager'
 difficulty: beginner
 tags: [Linux, containers, factories, foundries]
-description: 'This article contains information about how to use the Portenta X8 Manager.'
+description: 'Learn about the fundamentals of the Portenta X8 and how to use the Portenta X8 Manager.'
 author: Benjamin Dannegård
 hardware:
   - hardware/04.pro/board/portenta-x8
 software:
   - fioctl
-
 ---
 
 ## Overview
 
-The **Portenta** X8 is one of the most advanced boards available from Arduino. And with that comes some new concepts that are not standard for Arduino boards. In this article, we will go through some of the foundations of the Portenta X8 and help you understand how the board works and how you can benefit from its advanced features of this board. You will learn about FoundriesFactory® and how containers on the Portenta X8 work.
+The **Portenta X8** is one of the most advanced boards available from Arduino, introducing new concepts not typically found in other Arduino boards. In this article, we will cover the foundations of the Portenta X8, helping you understand how it works and how you can leverage its advanced features. You will learn about FoundriesFactory® and how containers on the Portenta X8 work.
 
 ## Goals
 
-- Get in-depth information about how the Portenta X8 works
-- Learn how containers work
+- Understand how the Portenta X8 works
+- Learn how containers work on the Portenta X8
 
 ### Required Hardware and Software
 
 - [Portenta X8](https://store.arduino.cc/portenta-x8)
+- [USB-C® cable (USB-C® to USB-A cable)](https://store.arduino.cc/products/usb-cable2in1-type-c)
 - [fioctl](https://docs.foundries.io/latest/getting-started/install-fioctl/index.html)
 
 ## Instructions
 
-If you need help setting up your board, please have a look at the [Portenta X8 User Manual](https://docs.arduino.cc/tutorials/portenta-x8/user-manual) where you will learn the basics and how to use your Portenta X8.
+If you need help setting up your board, please refer to the [**Portenta X8 User Manual**](https://docs.arduino.cc/tutorials/portenta-x8/user-manual) to learn the basics and get started with your Portenta X8.
 

```

Comment:
Before staring with the explanations of embedded linux, yocto etc,. It is worth mentioning what x8 is running: Yocto Distribution with support for docker containers.

Reply:
Updated!